src/components/Card.tsx

```diff
@@ -0,0 +1,13 @@
+interface ICardProps {
+  children: React.ReactNode;
```

Comment:
I noticed that you don't have a prop for the className, it would be good to have it so we can customize the card if we need.

```suggestion
  children: React.ReactNode;
  className?: string;
```